rpcs3/Emu/Cell/lv2/sys_sync.h

```diff
@@ -465,7 +465,7 @@ struct lv2_obj
 
 				// While IDM mutex is still locked (this function assumes so) check if the notification is still needed
 				// Pending flag is meant for forced notification (if the CPU really has pending work it can restore the flag in theory)
-				if (cpu != &g_to_notify && static_cast<const decltype(cpu_thread::state)*>(cpu)->none_of(cpu_flag::signal + cpu_flag::pending))
+				if (0) if (cpu != &g_to_notify && static_cast<const decltype(cpu_thread::state)*>(cpu)->none_of(cpu_flag::signal + cpu_flag::pending))
```

Comment:
Missing explanation.
Also, would probably look and read better if you simply wrote if (false &&